plugins/com.google.cloud.tools.eclipse.dataflow.ui/src/com/google/cloud/tools/eclipse/dataflow/ui/messages.properties

```diff
@@ -6,16 +6,16 @@ DATAFLOW_PIPELINE_OPTIONS=Set default options for running a Dataflow Pipeline.
 execution.options.for.google.cloud.platform=Execution Options for Google Cloud Platform
 CREATE_DATAFLOW_PROJECT=Create a Cloud Dataflow Project
 WIZARD_DESCRIPTION=This wizard creates a new Google Cloud Dataflow project.
-GROUP_ID_TOOLTIP=The Maven Group ID. Should be an alphanumeric path separated by periods.
-ARTIFACT_ID_TOOLTIP=The Maven Artifact ID. Should be an alphanumeric name separated by dashes.
+GROUP_ID_TOOLTIP=The Maven Group ID. Should Be an Alphanumeric Path Separated by Periods
+ARTIFACT_ID_TOOLTIP=The Maven Artifact ID. Should Be an Alphanumeric Name Separated by Dashes
```

Comment:
This casing is hard to read.  And we use sentence casing in our AE wizard.  Even m2e doesn't provide guidance on its labels.